README.md

```diff
@@ -20,8 +20,9 @@ The module performs the following tasks:
 <p align="center">
   <br/>
   <img src="hardware/Connection Diagram.svg">
+  <img src="hardware/Module Image.jpg">
```

Comment:
Why not using pure markdown? It should be as easy as this:
```markdown
![module-image](hardware/Module%20Image.jpg)
```
Plus it adds margin between the connection diagram and the image.

Reply:
I will use it.